### ISSUE TITLE
Tree Component Accessibility Issue with Role on Div instead of li element

### ISSUE THREAD

**Issue by technoligest** [primefaces/primereact#1004]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://forum.primefaces.org/viewforum.php?f=57
```

**Current behavior**
Tree component has an accessibility issue where
there is Role=Treeitem on the div element

**Expected behavior**
role=treeitem should be on the li element

**Minimal reproduction of the problem with instructions**
Create any Tree component

* **React version:**
16.8.6

* **PrimeReact version:**
3.1.2

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
ALL
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
Any

**Comment by mertsincan** [comment]
For screenreaders, I think the role property should be in the focused element. Could you please try it with screenreader such as JAWS, NVDA etc.?

**Comment by mertsincan** [comment]
Hi,

In 7.0 version, we will focus entirely on improving accessibility of components. Roadmap;
https://github.com/primefaces/primereact/wiki/Roadmap

I've closed this issue for now ;) If the problem persists after 7.0, please reopen this issue.

Best Regards,
